architecture/2025-04-aci-confidential-containers.md

```diff
@@ -0,0 +1,388 @@
+# Radius Integration with Azure Container Instances (ACI) - Confidential Containers
+
+* **Author**: Shruthi Kumar (@sk593)
+
+## Overview
+
+The integration of Azure Container Instances (ACI) with Radius aims to enable deployments of applications and containers using the Radius control plane in Kubernetes to target ACI environments. One of the key features we want to implement is the use of confidential containers to enhance data security and privacy. Confidential containers on Azure provide a set of features and capabilities to further secure your standard container workloads. They run in a hardware-backed Trusted Execution Environment (TEE) that provides intrinsic capabilities like data integrity, data confidentiality, and code integrity.
```

Comment:
Are we just going to deploy applications and containers? Anything else we can deploy with ACI but will do it later in future iterations?

Reply:
ACI is for serverless containers so it'll just be containers for now